Dockerfile

```diff
@@ -1,15 +1,9 @@
-FROM quay.io/armosec/kubescape
+FROM quay.io/kubescape/kubescape:dev-v2.0.371
 
 USER root
+COPY entrypoint.sh /entrypoint.sh
 
-RUN addgroup -S non-root && adduser -S non-root -G non-root
-
-RUN chown -R non-root:non-root /home/non-root
-
-USER non-root
-
-WORKDIR /home/non-root
-
-ADD entrypoint.sh /entrypoint.sh
+ENV KS_SKIP_UPDATE_CHECK true
+ENV KS_DOWNLOAD_ARTIFACTS false
 
 ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
Execute file from the local directory and not root directory (`"./entrypoint.sh"`)